scripts/config/msu/classes/enum.nut

```diff
@@ -0,0 +1,49 @@
+::MSU.Class.Enum <- class
+{
+	_Keys = null;
```

Comment:
Afaik, we don't prefix class members with `_` so far in MSU. Do we want to start this convention now? My personal opinion is that it is unnecessary as by default we keep all members private and only expose them via getter setter functions as desired.

Reply:
I think we should start the convention to do that yes, (for private members specifically) also in this case it means that there are no keys that are not allowed in the enum.

Reply:
So far, we've had no public member in any class and I don't anticipate it happening any time soon. We use only getter/setter functions. This is the recommended way anyway. Public members are boilerplate and should be avoided. Public Properties in C# are getters/setters too. Therefore, I see this as redundant. All members should be private. Expose the ones you want to be public via getters/setters.

For functions, I agree that private ones should have a prefix.

Reply:
@TaroEld disagrees with what you're saying here, therefore I'd rather be as explicit as possible

Reply:
I understand that not having the prefix doesn't make it immediately obvious if this member is publicly accessible or not (via separately defined getter/setter functions) unless you check for the getter/setter functions' existence. However, the issue with your approach is that if tomorrow you want to expose any of these private members via a getter/setter function, you'd end up having to refactor a bunch of lines to remove the `__` prefix from the variable name.

Reply:
Yes, there's not other way to do it since don't have a private function marker in squirrel. I don't see why this is an issue I think it's clean and standardized like this.